src/diffusers/loaders/textual_inversion.py

```diff
@@ -453,3 +453,91 @@ def load_textual_inversion(
             self.enable_sequential_cpu_offload()
 
         # / Unsafe Code >
+
+    def unload_textual_inversion(
+        self,
+        tokens: Optional[Union[str, List[str]]] = None,
+    ):
+        r"""
+        Unload Textual Inversion embeddings from the text encoder of [`StableDiffusionPipeline`]
+
+        Example:
+        ```py
+        from diffusers import AutoPipelineForText2Image
+        import torch
+
+        pipeline = AutoPipelineForText2Image.from_pretrained("runwayml/stable-diffusion-v1-5")
+
+        # Example 1
+        pipeline.load_textual_inversion("sd-concepts-library/gta5-artwork")
+        pipeline.load_textual_inversion("sd-concepts-library/moeb-style")
+
+        # Remove all token embeddings
+        pipeline.unload_textual_inversion()
+
+        # Example 2
+        pipeline.load_textual_inversion("sd-concepts-library/moeb-style")
+        pipeline.load_textual_inversion("sd-concepts-library/gta5-artwork")
+
+        # Remove just one token
+        pipeline.unload_textual_inversion("<moe-bius>")
+        ```
+        """
+
+        tokenizer = getattr(self, "tokenizer", None)
+        text_encoder = getattr(self, "text_encoder", None)
+
+        # Get textual inversion tokens and ids
+        token_ids = []
+        last_special_token_id = None
+
+        if tokens:
+            if isinstance(tokens, str):
+                tokens = [tokens]
+            for token_id, added_token in tokenizer.added_tokens_decoder.items():
```

Comment:
Shouldn't we only try to remove the specific `tokens` if it is not `None`?
i.e. we would try to find `token_id` of the `tokens` user passed (e.g.`for token in tokenizer(token)["input_ids"] ...`) instead of looping through `added_token_decoder.items()`

Reply:
yes, we can retrieve the token_id with tokenizer(token)["input_ids"][1]. 
I was looping through `added_token_decoder.items()` to avoid removing non-Textual Inversion tokens and special tokens

Reply:
```suggestion
            for added_token_id, added_token in tokenizer.added_tokens_decoder.items():
```